tests/grid/tests.h

```diff
@@ -0,0 +1,78 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2019 - 2022 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_grid_tests_h
+#define dealii_grid_tests_h
+
+#include "../tests.h"
+
+
+template <int dim, int spacedim>
+void
+print_statistics(const Triangulation<dim, spacedim> &tria, bool do_mg = false)
```

Comment:
It seems that you have not removed these function from the `fullydistributed_grid` folder!?